MAbdurrehman12/edx-platform
# fix: In Studio, ALLOW_PUBLIC_ACCOUNT_CREATION flag does not show/hide signup section above footer.

## Description

ALLOW_PUBLIC_ACCOUNT_CREATION is a flag in edx-platform -> cms -> envs -> common.py, which when set to **True** shows the signup button in navbar on studio page, else it hides the signup button. However, it does not hide a signup section above footer. Which means user can still sign up with  ALLOW_PUBLIC_ACCOUNT_CREATION flag set to **False**. 

Signup button which does not show when this flag is set to False: 
<img width="1438" alt="studio navbar" src="https://user-images.githubusercontent.com/93309234/139846891-0350f27c-41c0-4881-aba1-8e5080e87ea0.png">

Signup section which still shows when this flag is set to False: 
<img width="1426" alt="Signup section above footer" src="https://user-images.githubusercontent.com/93309234/139846534-2690a667-053f-4384-849b-6588203df3b6.png">

So in this PR, the condition applied to the signup button in navbar has also been applied to the signup section above footer, hence the issue is fixed.

## Supporting information
None

## Testing instructions

Set ALLOW_PUBLIC_ACCOUNT_CREATION to **True** and you will see both, signup button in navbar and signup section above footer.

Set ALLOW_PUBLIC_ACCOUNT_CREATION to **False** and both will be hidden.

## Deadline
None

## Other information
None
